runtime/moonbeam/src/lib.rs

```diff
@@ -156,7 +156,7 @@ pub mod currency {
 
 /// Maximum weight per block
 pub const MAXIMUM_BLOCK_WEIGHT: Weight = Weight::from_parts(WEIGHT_REF_TIME_PER_SECOND, u64::MAX)
-	.saturating_div(2)
+	.saturating_mul(1)
```

Comment:
```suggestion
```

Reply:
It's completely unnecessary to multiply by one, I know, I onyl replicated what's in moonriver runtime. Maybe it's fine to keep it as a guide (then when raising limits is only changing a number).